Twig/DoctrineExtension.php

```diff
@@ -39,7 +39,8 @@ public function getFilters()
         return array(
             new \Twig_SimpleFilter('doctrine_minify_query', array($this, 'minifyQuery')),
```

Comment:
given that the panel does not minify queries anymore, this filter should be marked as deprecated (Twig has a feature for that)